<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting_EasyQuat</class>
 <widget class="QDialog" name="Setting_EasyQuat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="CB_sed_inputs">
           <property name="text">
            <string>Read sequence-inputs file</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLineEdit" name="line_seq_inputs">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PB_Load_seq_inputs">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_12">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Simulation parameters</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_starttime">
              <property name="font">
               <font>
                <weight>50</weight>
                <italic>true</italic>
                <bold>false</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Start time</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>51</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="PB_load_fr_Sce">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Same as Sce</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_MJDdate_st">
                  <property name="text">
                   <string>MJD date:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="line_start_MJDdate">
                  <property name="maximum">
                   <number>999999</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>18</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_MJDsec_st">
                  <property name="text">
                   <string>MJD sec:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="line_start_MJDsec">
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>86400.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>18</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_Endtime">
              <property name="font">
               <font>
                <italic>true</italic>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>End time</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_MJDdate_end">
                <property name="text">
                 <string>MJD date:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="line_end_MJDdate">
                <property name="maximum">
                 <number>999999</number>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>18</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label_MJDsec_end">
                  <property name="text">
                   <string>MJD sec:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="line_end_MJDsec">
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>86400.000000000000000</double>
                  </property>
                  <property name="singleStep">
                   <double>0.100000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>18</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="4" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_Stepsize">
                <property name="font">
                 <font>
                  <italic>true</italic>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Step size</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="line_stepsize">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_sec">
                <property name="text">
                 <string>(sec)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_13">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Initial condition</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_Stepsize_2">
                <property name="text">
                 <string>Quaternion:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_sec_2">
                <property name="text">
                 <string>(w + xi + yj + zk)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QDoubleSpinBox" name="line_quat_w">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="line_quat_x">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="line_quat_y">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="line_quat_z">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_14">
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Spinning rate</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_Stepsize_3">
                <property name="text">
                 <string>Angular Velocite:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_sec_3">
                <property name="text">
                 <string>(xi + yj + zk)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QDoubleSpinBox" name="line_ang_x">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-999.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="line_ang_y">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-999.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="line_ang_z">
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="minimum">
                 <double>-999.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>999.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_sec_4">
                <property name="text">
                 <string>(rad/s)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Output location:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLineEdit" name="line_output_location">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PB_output_location">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Change</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="PB_OK">
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_cancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_clearall">
           <property name="text">
            <string>Clear all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CB_sed_inputs</tabstop>
  <tabstop>line_seq_inputs</tabstop>
  <tabstop>PB_Load_seq_inputs</tabstop>
  <tabstop>PB_load_fr_Sce</tabstop>
  <tabstop>PB_OK</tabstop>
  <tabstop>PB_cancel</tabstop>
  <tabstop>PB_clearall</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
